code/style.css

```diff
@@ -1,7 +1,322 @@
+  line-height: 1.5em;
+  padding-left: 3em;
+  font-size: 20px;
+}
+
+video#bgvideo {
+  position:fixed;
+  top: 50%;
+  left: 50%;
+  min-width: 100%;
+  min-height: 100%;
+  width: auto;
+  height: auto;
+  z-index: -100;
+  -ms-transform: translateX(-50%) translateY(-50%);
+  -moz-transform: translateX(-50%) translateY(-50%);
+  -webkit-transform: translateX(-50%) translateY(-50%);
+  transform: translateX(-50%) translateY(-50%);
+  /* background: url(placeholder.jpg) no-repeat; */
+  background-size: cover;
+}
+
+/* This is to create space before the forms and help with the free-flow feeling */
+.none{
+  padding: 2em;
+  background-color: transparent;
+  }
+
+/* Section underneath the form and the accordion   */
+.two{
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  flex-wrap: nowrap; 
+  gap: 3em;
+  padding: 1em 7em 10em 7em;
+  background-color: transparent;
+}
+
+.form{
+  border-style: none;
+  padding: 2em;
+  text-align: left;
+  background-color: #B8CABF;
+  border-radius: 0.5em;
+  font-family: 'Work Sans', sans-serif;
+  opacity: 80%;
+}
+.form:hover{
+  opacity: 100%;
+}
+
+input{
+  border: solid;
+  border-width: 1px;
+  font-size: 1em;
+  padding: 1em;
+  margin-bottom: 1em;   
+}
+
+.textblock{
+  width: 90%;
+}
+
+input:valid{
+  box-shadow:0.25em 0.25em 0.25em rgb(76, 205, 76);
+}
+
+input:invalid{
+  box-shadow: 0.2em 0.2em 0.2em rgb(232, 89, 89);
+}
+
+.label {
+  display: block;
+  padding: 0.5em;
+  text-align: left;
+}
+.name{
+  border-style: solid;
+  text-align: left;
+}
+
+.button{
+  margin: 1em;
+  padding: 1em;
+  border-radius: 1em;
+  background-color: #F5D081;
+}
+
+.accordion{
+  border-style: solid;
+  padding: 2em;
+  width: 70%;
+  font-family: 'Work Sans', sans-serif;
+  font-size: 25px;
+  background-color: #B8CABF;
+  border-radius: 0.5em;
+  opacity: 80%;
+}
+
+.accordion:hover{
+  opacity: 100%;
+}
 
 .active + .answer {
-  /* some styling here */
+  max-height: 5em;
+  font-style: italic;
+  font-size: 18px;
+}
+  
+.active {
+   font-weight: bold;
+}
+  
+.answer{
+   overflow: hidden;
+   max-height: 0;
+   transition: 1s ease-in-out;
+  }  
+
+.question:after {
+    /* Unicode character for "plus" sign (+) */
+    content: '\002B';
+    color: #19294C;
+    float: right;
+  }
+
+.active:after {
+  /* Unicode character for "minus" sign (-) */
+  content: "\2212";
+  color: #19294C;
+}
+
+/* This is the third section where a row of images appear */
+.img-container {
+    display: grid;
+    flex-wrap: wrap;
+    grid-template-columns: repeat(3, 1fr);
+  }
+    
+.grid-item-img {
+    grid-column: span 1;
+    display: grid;
+    flex-direction: column;
+  }
+  
+footer{
+    display: flex;
+    font-family: 'Work Sans', sans-serif;
+    justify-content:space-between;
+    flex-wrap: wrap;
+    align-items: center;
+    background-color: #6A8187 ;
+    padding: 5em;
+    color: white;
+    font-size: 20px;
+  }
+  
+@media screen and (max-device-width: 640px) {
```

Comment:
I see that you didn't start with mobile first. Maybe you might want to try to do a mobile-first approach on your next project?

Reply:
Great observation, that is my plan for sure! :) Feel comfortable now with the basic code so I will start the other way.